assets/pages/oneoff-contributions/oneoffContributions.jsx

```diff
@@ -27,6 +28,10 @@ const countryGroup = detectCountryGroup();
 
 const store = pageInit(reducer(getAmount('ONE_OFF', countryGroup)), true);
 
+const ONE_OFF_CONTRIBUTION_COOKIE = 'gu.contributions.contrib-timestamp';
```

Comment:
I found this a bit confusing because the cookie name suggests it is generic, not unique to one-off contributions.